docs/tutorials/10_Simulation_&_Tracking_Components.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+"""
+10 - Simulation & Tracking Components
+=====================================
+Running through the tutorials, the simulation code and tracking code has developed as new elements
+have been added. In the last tutorial, this ended with a simulation of varying number of targets,
+and a multi-target tracking handling initialisation and deletion of tracks. This tutorial just
+wraps up these elements by showing the Stone Soup components that implement the similar simulation
+and tracking features in existing components.
+"""
+
+# %%
+# Creating Simulation
+# -------------------
+# First create models as seen in previous tutorials
+import numpy as np
+
+from stonesoup.models.transition.linear import (
+    CombinedLinearGaussianTransitionModel, ConstantVelocity)
+from stonesoup.models.measurement.linear import LinearGaussian
+
+transition_model = CombinedLinearGaussianTransitionModel(
+    [ConstantVelocity(0.05), ConstantVelocity(0.05)])
+measurement_model = LinearGaussian(4, [0, 2], np.diag([0.25, 0.25]))
+
+# %%
+# And with those models, create a multi target ground truth simulator, feeding into a simple
+# detections simulator. These have a number of configurable parameters: defining where tracks are
+# born and at what rate, death probability, and where clutter is generated and at what rate, and
+# detection probability, etc.
+import datetime
+
+from stonesoup.simulator.simple import MultiTargetGroundTruthSimulator, SimpleDetectionSimulator
+from stonesoup.types.array import StateVector, CovarianceMatrix
+from stonesoup.types.state import GaussianState
+
+groundtruth_sim = MultiTargetGroundTruthSimulator(
+    transition_model=transition_model,
+    initial_state=GaussianState(
+        StateVector([[0], [0], [0], [0]]),
+        CovarianceMatrix(np.diag([4, 0.5, 4, 0.5]))),
+    timestep=datetime.timedelta(seconds=5),
+    number_steps=20,
+    birth_rate=0.3,
+    death_probability=0.05
+)
```

Comment:
I think it may be worth adding code snippets which show the original code which is now wrapped up here (or pointing to the specific tutorial section). That would allow the reader to directly compare without having two files open

Reply:
I've added links to sections (note these will be broken until #238 is also merged.